### ISSUE TITLE
[NGO profile] No labels explaining the financial reports (tooltip)

### ISSUE THREAD

**Issue by tudoramariei** [code4romania/onghub#351]
**Describe the bug**
Labels explaining the fields in a financial report don't appear. The HTML code seems to have this data available for the screen readers but not for other users.

Since this screen has a lot of specialized information, the labels should be available for all users, not just as an accessibility feature.

**Edit: The same label is present in all the texts. The translations.json files store the label in the same `financial.report_row` variable for all the `?` labels. Each field should have different labels; the items in `financial.income.*` should be extended from having just a string to having multiple properties (i.e. title, label, placeholder).**
(bolded because it's important to have different labels for users)

**Steps to reproduce the behavior**
1. Use a screen that allows for at least a 1650px wide window
2. Log in to an organization admin's account
3. Go to the `/organization/data` page
4. Open the `Financial info` tab
5. Edit one of the reports
6. Hover your mouse over the `?` label

**Expected behavior**
When hovering over a `?` label, a text explaining what the field means should appear.

**Screenshots**

![Screenshot 2023-09-29 at 11 17 18](https://github.com/code4romania/onghub/assets/2820084/6da8c113-9b72-40ce-8698-23a8e801fbe6)

**Device information**
 - OS: MacOS 13.5.2
 - Browser: Chrome 117.0.5938.92 [ARM]
